client/me/purchases/concierge-banner/index.jsx

```diff
@@ -0,0 +1,46 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import ActionCard from 'components/action-card';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+class ConciergeBanner extends Component {
+	componentDidMount() {
+		recordTracksEvent( 'calypso_purchases_concierge_banner_view', {
```

Comment:
i think this needs to be bound to dispatch to work?

Reply:
also check out the `<TrackComponentView />` React component which should be doing the same thing but without bleeding the internal details here

[README](https://github.com/Automattic/wp-calypso/blob/9b6b1e4b57ce3324850af20c913dcb03aca46aa6/client/lib/analytics/track-component-view/README.md)